README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Native Build Status](https://github.com/lancaster-university/microbit-v2-samples/actions/workflows/build.yml/badge.svg)](https://github.com/lancaster-university/microbit-v2-samples/actions/workflows/build.yml) [![Docker Build Status](https://github.com/lancaster-university/microbit-v2-samples/actions/workflows/docker-image.yml/badge.svg)](https://github.com/lancaster-university/microbit-v2-samples/actions/workflows/docker-image.yml)
 
-This repository is provides the tooling needed to compile a C/C++ CODAL program for the micro:bit v2 and produce a HEX file that can be downloaded to the device.
+This repository provides the necessary tooling to compile a C/C++ CODAL program for the micro:bit v2 and generates a HEX file that can be downloaded to the device.
```

Comment:
Thanks for the Pr @Swapnil-11-04-01!

For the last word `generates`, I believe it should be `generate` as it is meant to read "to compile ... and generate" not "to compile .. and generates".

Reply:
I'll apply this patch and merge, thanks @Swapnil-11-04-01! 
```suggestion
This repository provides the necessary tooling to compile a C/C++ CODAL program for the micro:bit v2 and generate a HEX file that can be downloaded to the device.
```

Reply:
Thanks, `@microbit-carlos`  for clarifying and patching the above. Also, I apologize for the late response, I am new in Open Source Contributions, therefore still learning and getting used to it.

Reply:
No worries, your contribution is appreciated, thanks for the PR!

Reply:
Thanks!